### PR TITLE
Koonom1985 patch atrts

### PR DESCRIPTION
**Pull request by koonom1985**

### CODE REVIEW

**Comment by koonom1985** [comment]
anyone help to check if this ATRTS is as expected?

**Comment by koonom1985** [comment]
this is for the issue #272

**Comment by twopirllc** [comment]
Hi @koonom1985,

This looks great on first glance. 😎  The _development_ branch has improved typing and a more common and consistent style. For example: [ssf](https://github.com/twopirllc/pandas-ta/blob/1b285814d72dcf8198889d75cf393ada110b1525/pandas_ta/overlap/ssf.py#L1) with _numba_ support. If you want give the new format a shot, awesome. If not, no worries.

Are you satisfied with the results so far? The core base indicators (**ma**, **atr**) are inline with TA Lib, so it should be fine.

Thanks,
KJ

**Comment by koonom1985** [comment]
sure, let me trytry the new format.

the result seems not very convincing as it shows diff with same of the example picture found online, not sure which part went wrong.

**Comment by koonom1985** [comment]
I double check on some ticker, it seems can imply the uptrend and downtrend.

**Review by twopirllc — APPROVED** [review]

**Comment by twopirllc** [comment]
Hi @koonom1985,

I have added it to the _development_ branch if you want to check it out and provide feedback.

```sh
$ pip install -U git+https://github.com/twopirllc/pandas-ta.git@development
```

Thanks
KJ

File: pandas_ta/volatility/atrts.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+    func_p = partial(calculateFunc, factor=factor)
+    atrts_ = [func_p(a,b,c,d) for a,b,c,d in zip(upTrend, dnTrend, prevP, atr_)]
+    atrts_ = Series(atrts_, index=close.index)
+
+    #prevA = atrts_.shift(1)
+    #atrts = [tailingStopFunc(a,b,c,d) for a,b,c,d in zip(upTrend, dnTrend, prevA, atrts_)]
+
+    #atrts = Series(atrts, index=close.index)
+    atrts = atrts_.shift(-1)
```
Comment (Spirulinea) [inline pandas_ta/volatility/atrts.py:134]: Wouldn't this lead to a lookahead bias?

Comment (twopirllc) [reply pandas_ta/volatility/atrts.py:134]: Nothing to worry about as the current [implementation](https://github.com/twopirllc/pandas-ta/blob/development/pandas_ta/volatility/atrts.py) is not shifting.

**Comment by wrone** [comment]
Hey @koonom1985 and @twopirllc, I'm testing this script and comparing the values to the Tradingview indicator - but I'm getting different results. I'm comparing it to "ATR Trailing Stoploss". 

I'm pretty new to this and probably doing something wrong, maybe I picked the wrong indicator, not sure. Maybe "ATR Trailing Stoploss" is not "ATR Trailing Stop"? 

Can someone please help me out with this?

**Comment by kmkswamy** [comment]
@wrone Did you find out the issue?

**Pull request closed without merging**